### ISSUE TITLE
[Instance - Zul'Farrak]Sergeant Bly

### ISSUE THREAD

**Issue by drpepper2** [TrinityCore/TrinityCore#11014]
Build 6cef507
DB: TDB 335.52

After attacking Sergeant Bly, all of his comrades go evade. 

(Part of Quest Divin-O-Matic Rod, http://www.wowhead.com/quest=2768)

**Comment by D4R4** [comment]
@tkrokli  Hey mate, I wanted to talk to you in private if that's possbile, sorry for spamming here, would you mind adding me on your skype? mine is "dorian.ardalan"

**Comment by tkrokli** [comment]
Just to remind myself about the core issue:
> @drpepper2 commented [on 8 Oct 2013](https://github.com/TrinityCore/TrinityCore/issues/11014#issue-20649439 "8. okt. 2013, 03:16 CEST")
After attacking Sergeant Bly, **all of his comrades go evade.**

**Comment by tkrokli** [comment]
Core : 4adf31c + tkrokli@073c14e (Win32, Release)
TDB 335.57 + updates + manual additions: 0c9e39f + 03edeeb

Re-tested on DK level 67 (used in other testing as well). None of the NPCs evade when turning hostile, but they are fairly static (unless you aggro them). Raven, Murta Grimgut and Oro Eyegouge will attack you if you are within aggro range (and haven't used the new settings **NoGrayAggro** on a much higher level character). I have tested with and without Mmaps enabled, it does not make a difference (except for that floating descent down the stairs by the NPCs in question, when Mmaps are disabled).

Worldserver.conf : mmap.enablePathFinding = 1
Worldserver.conf : mmap.enablePathFinding = 0

I also tested walking away from aggro range before Sergeant Bly has finished his exclamations. This makes it more distinct that Sergeant Bly is the only NPC who runs after your character after turning hostile, while the last 3 hostile NPCs (Raven, Murta Grimgut and Oro Eyegouge) stay perfectly still until you move your character within aggro range. Then they will attack normally for hostile NPCs.

Even though this hostile NPC behaviour is much less entertaining than it was live in 2010, the fact remains that the issue in the original post seems to be solved and can no longer be reproduced.

**Unless someone wants more details, documentation and testing of this issue, it is safe to say that the issue can no longer be reproduced. Time to close the issue.** (... or perhaps after a grace period?)
